clients/gen/common.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+SPEC_DIR="${SCRIPT_DIR}/../../airflow/api_connexion/openapi"
+
+OPENAPI_GENERATOR_CLI_VER=4.3.1
+GIT_USER=${GIT_USER:-apache}
+
+function gen_client {
+    lang=$1
+    shift
+    docker run --rm \
+        -v "${SPEC_DIR}/v1.yaml:/spec" \
```

Comment:
Can you parameterize the script so that it is possible to generate the client from a different specification file to a different directory?
I imagine that I can call three commands to then see the differences in generated clients
```
clients/gen/go.sh --spec-file "<(curl https://github.com/apache/airflow/blob/master/airflow/api_connexion/openapi/v1.yaml)". --out "/tmp/client-master"
clients/gen/go.sh --spec-file "../../airflow/api_connexion/openapi/v1.yaml)". --out "/tmp/client-pr-2222"
diff "/tmp/client-master" "/tmp/client-pr-2222"
```
This can facilitate reviews of changes in generated clients.